### PR TITLE
#375 migrate to Fontist::Install.from_hash API

### PR DESCRIPTION
**Pull request by CAMOBAP**
- #375

### CODE REVIEW

File: lib/asciidoctor/standoc/converter.rb [hunk 1]
```diff
@@ -89,11 +89,26 @@ def html_doc_path(file)
       end
 
       def flavor_name
```
Comment (CAMOBAP) [inline lib/asciidoctor/standoc/converter.rb:91]: @opoudjis @ronaldtse maybe you can suggest a better way here?

File: lib/asciidoctor/standoc/converter.rb [hunk 2]
```diff
@@ -89,11 +89,26 @@ def html_doc_path(file)
...
-        self.class.name.split("::")&.[](-2).downcase
+        self.class.name.split("::")&.[](-2).downcase.to_sym
       end
 
       def fonts_manifest
-        File.expand_path(File.join(@libdir, "../../metanorma/", flavor_name, "fonts_manifest.yaml"))
+        flavor = flavor_name
+        registry = Metanorma::Registry.instance
+        processor = registry.find_processor(flavor)
```
Comment (CAMOBAP) [inline lib/asciidoctor/standoc/converter.rb:98]: @opoudjis @ronaldtse maybe you can suggest a better way how to access related processor?

Comment (opoudjis) [reply lib/asciidoctor/standoc/converter.rb:98]: The processor is fine. processor.short == flavour_name, but that doesn't help you find the processor. Requiring the directory name to be the same as the flavour name (modulo capitalisation) is reasonable, and we are already presupposing it elsewhere.

**Comment by opoudjis** [comment]
Note that I am declining to merge today, in line with policy of not merging PRs I only discover on the day of release...

File: lib/asciidoctor/standoc/converter.rb (hunk 1 above)
Comment (ronaldtse) [reply lib/asciidoctor/standoc/converter.rb:91]: I don't have a better at the moment, but this is a good reminder that we need to update the interface...

File: lib/asciidoctor/standoc/converter.rb (hunk 2 above)
Comment (ronaldtse) [reply lib/asciidoctor/standoc/converter.rb:98]: Better Processor search interface needed...

**Review by ronaldtse — APPROVED** [review]

**Pull request merged**